### PR TITLE
Updated Utils class & Started TC145

### PR DESCRIPTION
**Pull request by Dimasss03**

### CODE REVIEW

File: src/utils/MainUtils.java [hunk 1]
```diff
@@ -2,4 +2,12 @@
 
 public class MainUtils {
 
+    public void coolMethod(){
```
Comment (VBobrusha) [inline src/utils/MainUtils.java]: cool methods but they should be static

**Comment by Dimasss03** [comment]
good catch , made them static

**Review by VBobrusha — COMMENTED** [review]
Great methods. looks good

**Review by VBobrusha — COMMENTED** [review]
good!

**Review by VBobrusha — COMMENTED** [review]
good good

**Review by VBobrusha — APPROVED** [review]

**Pull request merged**